src/inverseRobot.js

```diff
@@ -7,7 +7,16 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
-}
+  const InvRobot = {};
```

Comment:
please use camelCase
```suggestion
  const inversedRobot = {};
```